.github/test/test-workflows.js

```
@@ -0,0 +1,193 @@
+    
+    workflowFiles.forEach(file => {
+      const filePath = path.join(WORKFLOWS_DIR, file);
+      const content = fs.readFileSync(filePath, 'utf8');
+      const workflow = yaml.load(content);
+      
+      // Check if workflows that modify files have write permissions
+      if (file.includes('update') || file.includes('stats') || file.includes('badges')) {
+        const jobs = Object.values(workflow.jobs || {});
+        jobs.forEach(job => {
+          if (job.steps && job.steps.some(step => 
+            step.run && (step.run.includes('git commit') || step.run.includes('git push')))) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)